src/components/Avatar/styles.js

```diff
@@ -33,3 +33,30 @@ export const imgWrapper = css`
   justify-content: center;
   align-items: center;
 `
+export const marginBottom = css`
+  margin-bottom: 24px !important;
```

Comment:
I think !important can be removed here. CSS in js offers big specificity so there is no need for important rules